src/mudlet.cpp

```diff
@@ -478,7 +478,11 @@ mudlet::mudlet()
     connect(dactionIRC, &QAction::triggered, this, &mudlet::slot_irc);
     connect(dactionLiveHelpChat, &QAction::triggered, this, &mudlet::slot_irc);
 #if !defined(INCLUDE_UPDATER)
+    // Hide the update menu item if the code is not included
     dactionUpdate->setVisible(false);
+#else
+    // Also, only show it if this is a release version
+    dactionUpdate->setVisible(!scmIsDevelopmentVersion);
```

Comment:
This does mean that to test/debug things *now* one has to set the `APP_BUILD` project file variable to be an empty string to fake things so the code being tested thinks it is a release version...